src/EconomicStrategy/EconomicStrategyHelpers.ts

```diff
@@ -75,16 +75,14 @@ const isAboveMinBalanceLimit = async (config: Config, nextAccount: Address): Pro
  * Compares the profitability user settings and checks if the TimeNode
  * should claim a transaction.
  * @param {TransactionRequest} txRequest Transaction Request object to check.
- * @param {IEconomicStrategy} economicStrategy Economic strategy configuration object.
+ * @param {Config} config TimeNode configuration object.
  */
-const isProfitable = async (
-  txRequest: ITxRequest,
-  economicStrategy: IEconomicStrategy
-): Promise<boolean> => {
+const isProfitable = async (txRequest: ITxRequest, config: Config): Promise<boolean> => {
   const paymentModifier = await txRequest.claimPaymentModifier();
-  const reward = txRequest.bounty.times(paymentModifier);
+  const claimingGasCost = await config.util.estimateGas({});
```

Comment:
Is this a placeholder? What call is the gas estimation for?

Reply:
To deduct it from the reward estimation. It is used in the line below.

Reply:
Yeah but what exactly are you estimating the gas cost "of"?

Reply:
Claiming cost: https://github.com/ethereum-alarm-clock/timenode-core/issues/235.

Reply:
It should be specified in the object you pass to `estimateGas()`

see https://github.com/ethereum/wiki/wiki/JavaScript-API#web3ethestimategas

Reply:
Oops, didn't provide the tx as a parameter. Thanks.